extensions/2.0/Khronos/KHR_texture_float_bc6h/README.md

```diff
@@ -0,0 +1,219 @@
+﻿# KHR_texture_float_bc6h
+
+## Contributors
+
+* Rickard Sahlin, mailto:rickard.sahlin@ikea.com  
+* Alexey Knyazev [@lexaknyazev](https://github.com/lexaknyazev)  
+* Don McCurdy, Google, [@donrmccurdy](https://twitter.com/donrmccurdy)  
+* Norbert Nopper, mailto:nopper@ux3d.io  
+
+Copyright (C) 2021 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.  
+This extension requires the KHR_texture_basisu extension to be supported.  
```

Comment:
`KHR_texture_basisu` is unrelated to floating-point texture support.

Reply:
Yes it is, I have a hunch that transcoding may be needed to get support on targets that does not support BC6H.

Reply:
`KHR_texture_basisu` provides a fixed functionality of supporting ETC1S and UASTC codecs. Both of them are (and will remain) completely unrelated to BC6H.

Decoders for BC6H were developed separately and have no dependencies on Basis Universal.

Reply:
Yes, I know that but I thought that BC6H support was coming to basisu (meaning it could also be used as a transcoder should BC6H not be supported on target)?

Reply:
The basisu glTF extension is complete and therefore its functionality is frozen forever (except minor spec clarifications and guidelines).